src/medicine/main.py

```diff
@@ -1,47 +1,57 @@
-import pickle
-from icecream import ic
-from medicine.medicine_classes import Medicine
```

Comment:
Czemu mi usunąłeś icecream? :((

Reply:
bo nie jest potrzebny jak masz json

Reply:
Ja icecream używam do testowania i debugowania wstępnego

Reply:
ahh... To jest przydatne. Aczkolwiek nie powinno być bezpośrednio w kodzie. Kod, taki wypchany do gita, powinien być czysty. Tzn. używaj `icecream` wszędzie, tylko pamiętaj o tym żeby to usunąć przed wypchaniem do gita. To samo się tyczy `print` oraz `breakpoint` (które jest bardzo zacne - korzystaj często i gęsto).

Do tego warto utworzyć plik `requirements-dev.txt` gdzie będziesz miał bilblioteki pomocne podczas pisania, czyli `icecream`, `ipython`, `pdbpp` czy inne tego typu rzeczy. Wtedy jak twój projekt będzie instalowany gdzieś na poważnie (wiem, wiem), to będzie korzystało tylko z wymaganych narzędzi które są w `requirements.txt`. A jak któs będzie chciał móc milej pracować, to dodatkowe rzeczy zainstalują przez `pip install -r requirements-dev.txt`

Reply:
Fajne